qa/tasks/cephadm.py

```diff
@@ -1808,22 +1808,11 @@ def conf_epoch(ctx, config):
 def create_rbd_pool(ctx, config):
     if config.get('create_rbd_pool', False):
       cluster_name = config['cluster']
-      log.info('Waiting for OSDs to come up')
-      teuthology.wait_until_osds_up(
-          ctx,
-          cluster=ctx.cluster,
-          remote=ctx.ceph[cluster_name].bootstrap_remote,
-          ceph_cluster=cluster_name,
-      )
```

Comment:
why is it we don't want to wait for the OSDs to come up any more?

Reply:
I tried to explain it in the commit message:
- First, this code never worked because `teuthology.wait_until_osds_up()` wants to use adjust-ulimits wrapper which isn't available in "cephadm shell"
- Second, it's redundant because cephadm task waits for OSDs in `ceph_osds()`, which is called before this code is:
```
    with contextutil.nested(
            #if the cluster is already bootstrapped bypass corresponding methods
            lambda: _bypass() if (ctx.ceph[cluster_name].bootstrapped) \
                              else initialize_config(ctx=ctx, config=config),
            lambda: ceph_initial(),
...
            lambda: conf_setup(ctx=ctx, config=config),
            lambda: ceph_osds(ctx=ctx, config=config),
            lambda: ceph_mdss(ctx=ctx, config=config),
...
            lambda: ceph_clients(ctx=ctx, config=config),
            lambda: create_rbd_pool(ctx=ctx, config=config),
            lambda: conf_epoch(ctx=ctx, config=config),
    ):
```